gen/src/download/mod.rs

```diff
@@ -8,107 +8,86 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+
+//! UNIC - Data Download Script
+//!
+//! # Config Format:
+//!
+//! ```toml
+//! [example]
+//! version = "10.0.0"
+//! base_url = "http://example.com/path/to/{version}/"
+//!
+//! [[example.resources]]
+//! "some_data.txt" = "some_data.txt"
+//!
+//! [[example.test.resources]]
```

Comment:
This doesn't agree with the file format actually currently in use.

In use:
```toml
[example]
[example.resources]
[example.test_resources]
```

Shown here:
```toml
[example]
[[example.resources]]
[[example.test.resources]]
```

Reply:
Uh, good catch. I forgot to update it here.